pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -5000,76 +5371,158 @@ def TreatmentWaterQualityRule(b, r, qc, t):
     )
 
     def TreatedWaterQualityConcentrationBasedLHSRule(b, r, wt, qc, t):
-        constraint = (
-            b.v_Q[r, qc, t]
-            * (1 - b.parent_block().p_epsilon_TreatmentRemoval[r, wt, qc])
-            + b.parent_block().p_M_Concentration
-            * (
-                1
-                - sum(
+        if model.config.objective == Objectives.cost_surrogate:
+            if b.parent_block().p_chi_DesalinationSites[r]:
+                epsilon_value = 0.99
```

Comment:
Why is `epsilon_value` constrained to 0.99? Do the surrogate models assume that the rejection fraction of TDS is 99% or can it vary? Also, would this be affected if the user selects `RemovalEfficiencyMethod.load_based` vs `RemovalEfficiencyMethod.concentration_based`?

Reply:
Good catch, that should have been model.recovery instead of constant value of 0.99.

Reply:
Javal and I discussed this. @ElmiraShamlou can you please confirm whether the surrogate models assume 99% rejection of TDS?

Reply:
Both MVR and MD surrogate models assume 100% salt rejection, making a 99% removal efficiency an acceptable assumption in this context. However, we can consider this as an input parameter to provide flexibility for various scenario analyses, especially for possible future surrogates. It's important to leave a note in the model that the current MD and MVR surrogates were developed under the assumption of 100% removal efficiency.

Reply:
Thanks for the clarification @ElmiraShamlou.